### DISCUSSION TITLE
Nagni want fund $250 for looking some one to implement video player control buttons

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#1842] [Announcements]
RT, original post: #1811

**Comment by thienphuoc** [comment]
Has anyone addressed this issue yet? I have some experience handling native video controls, and I think I might be able to help.

**Comment by halx99** [comment]
Since the UIMediaPlayer is render videoTexure on axmol node-tree, cross-platform video control buttons preferred, refer PR: https://github.com/axmolengine/axmol/pull/1845
